util/autogen_testutils.py

```diff
@@ -36,13 +39,33 @@ def main():
             "ERROR: clang-format must be installed to format "
             " autogenerated code to pass OpenTitan CI checks.")
 
+    # Parse command line args.
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--topcfg", "-t", help="path of the top hjson file.")
+    args = parser.parse_args()
+
+    # Parse toplevel Hjson to get IPs that are templated / generated with IPgen.
+    topcfg_path = REPO_TOP / "hw/top_earlgrey/data/top_earlgrey.hjson"
```

Comment:
wouldn't it make more sense to use `default=` in the parser?

Reply:
Fixed.